packages/ckeditor5-dev-translations/lib/movetranslations.js

```diff
@@ -112,10 +113,38 @@ function assertContextsExist( { packageContexts, config } ) {
 		.map( entry => `Missing context: the "${ entry.messageId }" message does not exist in "${ entry.source }" package.` );
 }
 
+/**
+ * @param {MoveTranslationsOptions} options
+ */
+function normalizeOptions( options ) {
+	const { config } = options;
+
+	const cwd = options.cwd || process.cwd();
+	const toAbsolute = path => upath.resolve( cwd, path );
```

Comment:
```suggestion
	const toAbsolute = path => upath.join( cwd, path );
```

I think it fits our needs better as we want to merge two paths (an absolute and a relative). `resolve()` merges the specified arguments with `process.cwd()` if specified paths are relative. I know, it's an edge case here.

Reply:
We do not want just merge two paths. We want to calculate an absolute path from two arguments. The output from `toAbsolute()` must be an absolute path. The result from `upath.join()` is not always and absolute path (i.e. when both arguments are relative).

```
upath.join( '.', 'x/y/z' ) !== upath.resolve( '.', 'x/y/z' )
```

Using `process.cwd()` with `upath.resolve()` in case integrator did not specify own absolute `cwd` path is fine.

Reply:
Good point, I didn't think about passing a relative `cwd`.